translation/pt-BR/README.md

```diff
@@ -0,0 +1,367 @@
+Language: [English](../../README.md) | [Português](README.md)
+
+# mobx.dart
```

Comment:
Unfortunately, I don't understand Portuguese, so I will defer to your judgement for rest of the content :-)